setup.py

```diff
@@ -180,13 +180,25 @@ def get_outputs(self):
 )
 """
 
-package = 'tensorflow>=2.0.0,<2.1.0'
-version = '0.9.0'
-project = 'tensorflow-io'
+# read package and version from:
+# tensorflow_io/core/python/ops/io_info.py
```

Comment:
Why are we moving this to that file?

Reply:
@terrytangyuan We need `__version__` string in two places:
1) It needs to be in a python file shipped in wheel package so that in runtime it is exposed. 
2) It also need to be in setup.py so that it is inserted in wheel's egg.

However, setup.py will not be shipped by wheel package directly so we could not put the string in setup.py (we also don't want to place two places for the purpose of single source of truth).

So we move the string to this `tensorflow_io/core/python/ops/io_info.py` file:
1) In runtime, tensorflow_io/__init__.py will import the string `__version__`.
2) setup.py also needs the string when package the wheel file. In this case, we just read the file directly to grab the string and pass to setup.py.

Reply:
Got it